src/main/java/org/opendcs/database/impl/opendcs/OpenDcsPgProvider.java

```diff
@@ -0,0 +1,238 @@
+                              .invoke();
+                }
+            }
+        });
+    }
+
+    @Override
+    public List<File> getDecodesData()
+    {
+        List<File> files = new ArrayList<>();
+        String decodesData[] = {
+            "${DCSTOOL_HOME}/edit-db/enum",
+            "${DCSTOOL_HOME}/edit-db/eu/EngineeringUnitList.xml",
+            "${DCSTOOL_HOME}/edit-db/datatype/DataTypeEquivalenceList.xml",
+            "${DCSTOOL_HOME}/edit-db/presentation",
+            "${DCSTOOL_HOME}/edit-db/loading-app"};
+        fillFiles(files, decodesData, ".xml");
+        if (log.isTraceEnabled())
+        {
+            for (File f: files)
+            {
+                log.trace("Importing '{}'", f.getAbsolutePath());
+            }
+        }
+        return files;
+    }
+
+    @Override
+    public List<File> getComputationData()
+    {
+        List<File> files = new ArrayList<>();
+        String computationData[] = {
+            "${DCSTOOL_HOME}/imports/comp-standard/algorithms.xml",
+            "${DCSTOOL_HOME}/imports/comp-standard/Division.xml",
+            "${DCSTOOL_HOME}/imports/comp-standard/Multiplication.xml"
+            };
+        fillFiles(files, computationData, ".xml");
+        return files;
+    }
+
+    private void fillFiles(List<File> files, String fileNames[], String suffix)
+    {
+        for (String filename: fileNames)
+        {
+            File f = new File(EnvExpander.expand(filename, System.getProperties()));
+            if (f.exists() && f.canRead() && f.isFile())
+            {
+                files.add(f);
+            }
+            else if (f.exists() && f.canRead() && f.isDirectory())
+            {
+                try
+                {
+                    files.addAll(Files.walk(f.toPath())
+                                      .map(p -> p.toFile())
+                                      .filter(file -> file.isFile())
+                                      .filter(file -> file.getAbsolutePath().endsWith(suffix))
+                                      .collect(Collectors.toList())
+                        );
+                }
+                catch (IOException ex)
+                {
+                    log.atError()
+                       .setCause(ex)
+                       .log("Unable to process diretory '{}'", f.getAbsolutePath());
+                }
+            }
+            else if(f.exists() && !f.canRead())
+            {
+                log.error("File '{}' is not readable. Skipping.", f.getAbsolutePath());
+            }
+            else
+            {
+                log.error("File '{}' is not present. Skipping.", f.getAbsolutePath());
+            }
+        }
+    }
+
+    @Override
+    public void loadBaselineData(Profile profile, String username, String password) throws IOException
+    {
+        /* Initializing database with default data */
+        System.setProperty("DCS_PASS", username);
+        System.setProperty("DCS_USER", password);
```

Comment:
```suggestion
        System.setProperty("DCS_PASS", password);
        System.setProperty("DCS_USER", username);
```

Reply:
Yep. I've fixed it locally, there were other changes required as well to correct some issues.

Reply:
Also going to change the integration test user/name password combos. this would've been caught if I made sure to always make them different.